dream_layer_backend/test_api_standalone.py

```
@@ -0,0 +1,324 @@
+#!/usr/bin/env python3
+"""
+Standalone API test for report generation endpoints
+Tests the Flask API without requiring ComfyUI dependencies
+"""
+
+import os
+import sys
+import json
+import tempfile
+import shutil
```

Ruff (0.12.2)

`shutil` imported but unused

Remove unused import: `shutil`

(F401)